src/sentry/discover/arithmetic.py

```diff
@@ -65,12 +90,22 @@ class ArithmeticVisitor(NodeVisitor):
     DEFAULT_MAX_OPERATORS = 10
 
     # Don't wrap in VisitationErrors
-    unwrapped_exceptions = (MaxOperatorError,)
+    unwrapped_exceptions = (ArithmeticError,)
+
+    allowlist = {
```

Comment:
are we not allowing measurements in this pass?

Reply:
lol no i just forgot, thanks :+1: